test/e2e/lib/components/notices-component.js

```diff
@@ -15,18 +15,52 @@ import AsyncBaseContainer from '../async-base-container';
 
 export default class NoticesComponent extends AsyncBaseContainer {
 	constructor( driver ) {
-		super( driver, By.css( '#notices' ), null, config.get( 'explicitWaitMS' ) * 3 );
+		super( driver, By.css( '.wpcom-site' ), null, config.get( 'explicitWaitMS' ) * 3 );
 	}
 
-	async inviteMessageTitle() {
-		const selector = By.css( '.invites__title' );
+	async isSuccessNoticeDisplayed( click = false ) {
+		const selector = By.css( '.notice.is-success' );
+		const actionSelector = By.css( '.notice.is-success a' );
+		const isDisplayed = await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
```

Comment:
IIRC `waitTillPresentAndDisplayed` will throw a TimeoutError if the element is not found. `isSuccessNoticeDisplayed` name makes me think that this method will return boolean value instead of throwing an error. Maybe we can update method name or use `isEventuallyPresentAndDisplayed` to match behavior with method name? Maybe `waitForSuccessNotice`?

Reply:
Good catch! 👍